editor/plugins/script_text_editor.cpp

```diff
@@ -2370,6 +2444,29 @@ void ScriptTextEditor::drop_data_fw(const Point2 &p_point, const Variant &p_data
 	te->grab_focus();
 }
 
+void ScriptTextEditor::_assign_dragged_export_variables() {
+	ERR_FAIL_COND(dragged_exports_dirty.size() == 0);
+
+	Node *scene_root = get_tree()->get_edited_scene_root();
+
+	// Sanity check, should never be null
+	ERR_FAIL_COND(!scene_root);
+
+	Node *sn = _find_script_node(scene_root, script);
+	if (!sn) {
+		sn = scene_root;
+	}
+
+	ERR_FAIL_COND(sn->get_script_instance() == nullptr);
+
+	for (auto &dragged_export : dragged_exports_dirty) {
```

Comment:
```suggestion
	for (const DraggedExport &dragged_export : dragged_exports_dirty) {
```
We try to avoid auto.